### DISCUSSION TITLE
local device database?

### DISCUSSION THREAD

**Discussion by ahmedsweng** [ujjwalguptaofficial/JsStore#346] [Q&A]
Hello guys, I am new to NextJs I want to implement a local database for each user in my web app and also in the mobile app and I want to relate between the web and mobile with data backups to google drive and download the same data across the web and mobile how can I implement this architecture? the user should have the data locally and perform all the CRUD operations locally. can I do it with SQLite? anyone can help me?. Is JsStore the right choice?

**Comment by ujjwalguptaofficial** [comment]
JsStore is right choice for web but for mobile app - you will have to use sqlite or other database engines.
